### PR TITLE
For #15349: Fixes SyncedTabs suggestion not clicking (beta)

### PR DESCRIPTION
**Pull request by jonalmeida**
For v82.0.0 beta; identical to https://github.com/mozilla-mobile/fenix/pull/15395.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

**Review by ekager — APPROVED** [review]

**Comment by jonalmeida** [comment]
Probably best to verify this in nightly before we merge this PR? Unless we're cutting a new beta before that.

**Comment by jonalmeida** [comment]
Merging since QA has reported this is fixed in nightly: https://github.com/mozilla-mobile/fenix/issues/15349#issuecomment-699917653

**Pull request merged**
